stacks/SiriGeneratorStack.ts

```diff
@@ -8,14 +8,37 @@ export function SiriGeneratorStack({ stack }: StackContext) {
     const { disruptionsJsonBucket } = use(SiteStack);
 
     const siriSXBucket = new Bucket(stack, "SiriSXBucket", {
-        name: `cdd-siri-sx-${stack.stage}`,
+        name: `cdd-siri-sx-generator-${stack.stage}`,
```

Comment:
This should stay as `cdd-siri-sx-${stack.stage}`

Reply:
done